Student/Student/Quizzes/QuizDetails/QuizDetailsViewController.swift

```diff
@@ -162,6 +162,7 @@ class QuizDetailsViewController: ScreenViewTrackableViewController, ColoredNavVi
         let title = takeButtonTitle
         takeButton.setTitle(title, for: .normal)
         takeButton.isHidden = title == nil
+        takeButton.makeUnavailableInOfflineMode()
```

Comment:
The takeButton gets funky when this is applied, we might need to change its z order either in code or on the storyboard.
![IMG_5A058E9D8721-1](https://github.com/instructure/canvas-ios/assets/79920680/962d2683-53bc-4a43-aca2-687e33bb1ff7)